docs/install.rst

```diff
@@ -42,6 +42,7 @@ Adding To Your Django Project
         'south',  # Highly recommended to make database migrations simpler.
         'markdown_deux',  # Required for Knowledgebase item formatting
         'helpdesk',  # This is new!
+        'bootstrapform', 
```

Comment:
I guess we should add 'bootstrapform' before 'helpdesk', since it's a dependency.